### PR TITLE
alert user when car is about to disengage adaptive cruise

### PR DESCRIPTION
**Pull request by cydia2020**
Must be used together with https://github.com/commaai/cereal/pull/105

Minimise the effect of #2726

### CODE REVIEW

**Comment by grekiki** [comment]
Nothing wrong with the PR idea, but please mark as ready for review at least after passing the CI tests.

**Comment by cydia2020** [comment]
> Nothing wrong with the PR idea, but please mark as ready for review at least after passing the CI tests.

Hi, I am not sure but I think it failed because of commaai/cereal#105 . Is there a way to incorporate that PR when doing the checks?

**Comment by grekiki** [comment]
If you cd to openpilot/cereal you will get into another repository, where you can then checkout to your PR. Then if you go back to openpilot and run git status, you should see that the submodule was updated. Then you can commit and push, and it should hopefully pass the tests.

**Comment by cydia2020** [comment]
> If you cd to openpilot/cereal you will get into another repository, where you can then checkout to your PR. Then if you go back to openpilot and run git status, you should see that the submodule was updated. Then you can commit and push, and it should hopefully pass the tests.

Ah, thanks

File: selfdrive/car/toyota/interface.py [hunk 1]
```diff
@@ -340,6 +340,10 @@ def update(self, c, can_strings):
         # while in standstill, send a user alert
         events.add(EventName.manualRestart)
 
+    if ret.vEgo < 0.001 and self.CS.pcm_acc_status == 7:
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/interface.py:343]: doesn't the car show an alert? I don't like this alert creation delay hack

Comment (cydia2020) [reply selfdrive/car/toyota/interface.py:343]: No, the car only alerts when it reaches 180 seconds (and shift to p), in which case openpilot will panic as well for wrong gear.

**Comment by cydia2020** [comment]
closing until i find a less janky way of counting this.

**Pull request closed without merging**